soa/logger/kvp_logger_interface.cc

```diff
@@ -1,5 +1,4 @@
 #include "kvp_logger_interface.h"
```

Comment:
Discussion indicated that the changes in this file may no longer be required so let's keep the diff with master as small as possible to make future merges easier.